deploy/grafana-dashboard/general_row.py

```diff
@@ -0,0 +1,112 @@
+from grafanalib.core import RowPanel, GridPos, Stat, TimeSeries, Heatmap, BarGauge, GAUGE_DISPLAY_MODE_GRADIENT
+from grafanalib.influxdb import InfluxDBTarget
+
+from color_mapping import grey_all_mapping
+from util import read_query
+
+requests_per_minute = TimeSeries(
+    title="Requests per minute",
+    dataSource='Poseidon',
+    targets=[InfluxDBTarget(query=read_query("requests-per-minute"))],
+    gridPos=GridPos(h=9, w=8, x=0, y=1),
+    scaleDistributionType="log",
+    extraJson=grey_all_mapping
+)
+
+request_latency = Heatmap(
+    title="Request Latency",
+    dataSource='Poseidon',
+    dataFormat="timeseries",
+    targets=[InfluxDBTarget(query=read_query("request-latency"))],
+    gridPos=GridPos(h=9, w=8, x=8, y=1),
+    extraJson={
+        "options": {},
+        "yAxis": {
+            "format": "ns"
+        }
+    }
+)
+
+service_time = TimeSeries(
+    title="Service time (99.9%)",
```

Comment:
I was wondering whether we should allow more fine granular data points here (similar to the current dashboard version) to account for smaller differences.

Reply:
I have adjusted this dashboard to use the dynamic value again.